fabric/path_test.go

```diff
@@ -0,0 +1,31 @@
+package fabric
+
+import "testing"
+
+func TestFabricPathToPredicate(t *testing.T) {
```

Comment:
This test represents current Fabric behaviors as is. @szuecs Is there anything current behaviours that you would like to change/deprecate?

Reply:
I don't know, I think it should be the same. If you have a reason to change it let me know, but I guess there is no reason.

Reply:
I mainly think about poor documented `*named_match` cases. It only appeared to me by reading code and tests, not the docs. I plan to check the adoption of those.